### ISSUE TITLE
Hyperparameter choices

### ISSUE THREAD

**Issue by bkj** [melodyguan/enas#11]
Hi -- 

Are you able to explain a little how you selected the various hyperparameters?  I see that they are a) different from those mentioned in the paper and b) different between the `*_search` and `*_final` stages. 

~ Ben

**Comment by hyhieu** [comment]
It turns out that `*_search` can produce different architectures, so the hyper-parameters have to be re-tuned a little bit. We do this by setting `num_valids=5000` [at this line](https://github.com/melodyguan/enas/blob/master/src/cifar10/main.py#L219) and finally use the best hyper-parameters to run with `num_valids=0`.

**Comment by bkj** [comment]
Sorry -- not sure I understand.  You're saying that you

  - manually change that line to say `num_valids=5000`
  - manually tune hyperparameters to optimize validation accuracy
  - run the tuned configuration w/ `num_valids=0`
  - report test accuracy

**Comment by hyhieu** [comment]
Yes. I think your understanding is correct.
